client/homebrew/pages/editPage/editPage.jsx

```diff
@@ -91,6 +142,14 @@ const EditPage = createClass({
 		document.removeEventListener('keydown', this.handleControlKeys);
 	},
 
+	editTypeIsNew : function() {
```

Comment:
Can you come up with a shorter name for these? The purpose of a function like this is essentially shorthand for `this.props.editType == 'edit'`, but with a longer name like this it's not that much shorter. I might actually prefer just checking `this.props.editType == 'edit'` directly because it's a bit more straightforward to me.

Reply:
Yes, it can absolutely be renamed to something shorter. `typeNew()`, `typeEdit()` or something to that effect.

Reply:
I do like the `is` style for straight boolean checkers like this. How about `isNew()` and `isEdit()`?

Reply:
I've just pushed through a new commit where the functions are renamed to `isNew()` and `isEdit()`.